### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by rartych**
Codeowners from Telefonica added

#### What type of PR is this?

Add one of the following kinds:
* subproject management

#### What this PR does / why we need it:

Codeowners from Telefonica added.

### CODE REVIEW

**Comment by MarkusKuemmerle** [comment]
Please make sure that the new codeowners are part of the Sub Project before merging this PR!
Both Ruben and José are not it in the moment.

**Comment by PedroDiez** [comment]
Hi, in order to be able to write access to this repo Markus has commented to me to be also codeowner.
So adding myself.

File: CODEOWNERS
Comment (PedroDiez) [inline CODEOWNERS:8]:
```suggestion
* @shilpa-padgaonkar @rartych @patrice-conil @RubenBG7 @jordonezlucena @PedroDiez
```

Comment (rartych) [reply CODEOWNERS:8]: Have you followed the instructions:
_To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit https://lists.camaraproject.org/g/sp-com._ ?

**Review by patrice-conil — APPROVED** [review]
LGTM

**Pull request merged**
